### DISCUSSION TITLE
[Feature] Remove faces permanently and manually tag faces

### DISCUSSION THREAD

**Discussion by dafeist-github** [immich-app/immich#7441] [Feature Request]
### The feature

A button to remove (wrongly detected) faces completely(from both "photos" and "people" pages), manually assign photos to people which the Face Detection has not detected

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by OverHash** [comment]
Related: #6205, [#2472](https://github.com/immich-app/immich/discussions/2472#discussioncomment-6850486)
